package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "resin-semver",
   "version": "0.1.1",
-  "description": "resin specific semver utility methods",
+  "description": "Resin.io specific semver utility methods",
   "main": "src/index.js",
   "typings": "src/index.d.ts",
   "scripts": {
     "test": "mocha -r ts-node/register --outDir tmp test/index.spec.ts",
-    "build": "tsc src/index.ts"
+    "build": "tsc src/index.ts",
+    "ci": "npm run test"
```

Comment:
I'm adding a lint script in a follow up pr, so the `ci` script will become `npm run lint && npm test`.